packages/cozy-harvest-lib/src/components/KonnectorSuggestionModal/DataTypes.jsx

```diff
@@ -0,0 +1,43 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import { translate } from 'cozy-ui/transpiled/react/I18n'
+import Text from 'cozy-ui/transpiled/react/Text'
+import CompositeRow from 'cozy-ui/transpiled/react/CompositeRow'
+import Icon from 'cozy-ui/transpiled/react/Icon'
+
+const DataTypes = ({ t, dataTypes, konnectorName }) => {
+  return dataTypes.length > 0 ? (
+    <>
+      <Text className="u-ta-center">
```

Comment:
Do we already have a stylus loader for harvest? We should try to not use u-* anymore